src/rhash/hash.c

```diff
@@ -1250,7 +1295,16 @@ static int rc_hash_nintendo_disc(char hash[33], const rc_hash_iterator_t* iterat
     {
       success = rc_hash_wii(&md5, iterator, file_handle);
     } else {
-      success = rc_hash_iterator_error(iterator, "Not a supported Nintendo disc");
+      /* Check WiiWare */
+      rc_file_seek(iterator, file_handle, 0x04, SEEK_SET);
+      rc_file_read(iterator, file_handle, quad_buffer, 2);
+      if (quad_buffer[0] == 'I' && quad_buffer[1] == 's')
+      {
+        success = rc_hash_wiiware(&md5, iterator, file_handle);
+      }
+      else {
+        success = rc_hash_iterator_error(iterator, "Not a supported Nintendo disc");
+      }
```

Comment:
make sure to add an entry for `wad` files in the `rc_hash_iterator_ext_handlers` table.